src/js/app.js

```diff
@@ -0,0 +1,103 @@
+import Digits from './components/Digits.js';
+import Total from './components/Total.js';
+import Operations from './components/Operations.js';
+import Modifiers from './components/Modifiers.js';
+import { $ } from './utils/dom.js';
+import { isOperation } from './utils/validate.js';
+import { DOM, OPERATION, MODIFIER, INIT_STATE, MESSAGE } from './constants.js';
+
+class App {
+  constructor(target) {
+    this.$target = $(target);
+    this.state = {
+      currentTotal: INIT_STATE.currentTotal,
+      numberCount: INIT_STATE.numberCount,
+    };
```

Comment:
```suggestion
    this.state = { ...INIT_STATE };
```
이렇게 쓰지 않으시는 이유는 INIT_STATE가 this.state와 모양이 다를 수도 있기 때문일까요?